stdlib/public/SwiftShims/swift/shims/EmbeddedShims.h

```diff
@@ -28,8 +28,7 @@
 extern "C" {
 #endif
 
-// FIXME: Replace with __has_feature(swiftcc) once that's added to Clang.
-#if __has_feature(swiftasynccc)
+#if __has_extension(swiftcc)
```

Comment:
Should we use `__has_feature(swiftasynccc) || __has_extension(swiftcc)` to be compatible with older compilers for a while?

@compnerd Do all windows builds use a just built compiler such that this is not a concern?

Reply:
I think it's worth being conservative here.